app/models/extra_mobile_data_request.rb

```diff
@@ -16,6 +16,7 @@ class ExtraMobileDataRequest < ApplicationRecord
   validates :agrees_with_privacy_statement, inclusion: { in: [true] }
 
   validate :validate_school_or_rb_present
+  validate :validate_request_uniqueness, on: :create
```

Comment:
There are some existing duplicates so we need to relax the constraint on update - otherwise MNOs they won't be able to update them at all (e.g., to update the status.)